src/main/java/net/atos/zac/app/productaanvragen/ProductaanvragenRESTService.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Atos
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+package net.atos.zac.app.productaanvragen;
+
+import static net.atos.zac.policy.PolicyService.assertPolicy;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.util.List;
+import java.util.UUID;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import org.apache.commons.collections4.CollectionUtils;
+
+import net.atos.client.zgw.drc.DRCClientService;
+import net.atos.client.zgw.drc.model.EnkelvoudigInformatieobject;
+import net.atos.zac.aanvraag.InboxProductaanvraagService;
+import net.atos.zac.aanvraag.model.InboxProductaanvraagListParameters;
+import net.atos.zac.aanvraag.model.InboxProductaanvraagResultaat;
+import net.atos.zac.app.productaanvragen.converter.RESTInboxProductaanvraagConverter;
+import net.atos.zac.app.productaanvragen.converter.RESTInboxProductaanvraagListParametersConverter;
+import net.atos.zac.app.productaanvragen.model.RESTInboxProductaanvraag;
+import net.atos.zac.app.productaanvragen.model.RESTInboxProductaanvraagListParameters;
+import net.atos.zac.app.productaanvragen.model.RESTInboxProductaanvraagResultaat;
+import net.atos.zac.app.shared.RESTResultaat;
+import net.atos.zac.policy.PolicyService;
+
+@Singleton
+@Path("productaanvragen")
+@Consumes(MediaType.APPLICATION_JSON)
+@Produces(MediaType.APPLICATION_JSON)
+public class ProductaanvragenRESTService {
```

Comment:
Hernoemen naar InboxProductaanvragenRESTService

Ik had hier dezelfde 3 methodes verwacht als in productaanvragen-service.ts: list, delete en getPDFViewerURL. Met exact dezelfde naam

Reply:
sub-task aangemaakt